src/NLog.Layouts.GelfLayout/GelfConverter.cs

```diff
@@ -112,7 +118,8 @@ public void ConvertToGelfMessage(JsonWriter jsonWriter, LogEventInfo logEventInf
                         continue;
 
                     foundMdlcItem = true;
-                    object propertyValue = MappedDiagnosticsLogicalContext.GetObject(key);
+                    object propertyValue;
+                    ScopeContext.TryGetProperty(key, out propertyValue);

```

Comment:
Maybe enumerate the result returned from `ScopeContext.GetAllProperties()` instead of doing `ScopeContext.TryGetProperty` ?